README.md

```diff
@@ -39,40 +39,44 @@ As an abstraction, this tool allows for greater consistency and maintainability
 
 ## Overview
 
-At its core, RESTful React exposes a component, called `Get`. This component retrieves data, either on mount or later, and then handles error states, caching, loading states, and other cases for you. As such, you simply get a component that _gets stuff_ and then does stuff with it. Here's a quick overview what it looks like.
+At its core, `restful-react` exposes a [hook](https://reactjs.org/docs/hooks-intro.html), called `useGet`. This component retrieves data, either on mount or later, and then handles error states, loading states, and other cases for you. As such, you get a component that _gets stuff_ and then does stuff with it. Here's a quick overview what it looks like.
 
-[![Edit Restful React demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/30n66z45mq)
+[![Edit restful-react demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/restful-react-demos-vjets)
```

Comment:
This Demo on Codesandbox is broken

<img width="868" alt="Screenshot 2019-10-15 at 15 34 06" src="https://user-images.githubusercontent.com/639406/66836260-4fba8200-ef61-11e9-8587-0dfdac508351.png">

Reply:
Fixed~